helix-term/src/commands/typed.rs

```diff
@@ -1012,6 +1012,37 @@ fn sort_impl(
     Ok(())
 }
 
+fn reflow(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    _event: PromptEvent,
+) -> anyhow::Result<()> {
+    // TODO: Can we instead take an Option<Cow<str>>, and then if the user
+    // doesn't pass in a selection, default to 79 characters per line?
+
+    const DEFAULT_MAX_LEN: Cow<'static, str> = Cow::Borrowed("79");
+    let max_line_len: usize = args.get(0).unwrap_or(&DEFAULT_MAX_LEN).parse()?;
```

Comment:
Maybe not necessarily in this PR, but it might also be cool to add a config option to set the default max line width if you prefer a wider width, but don't want to give it explicitly every time.

Reply:
Agreed. I can add that real quick to this PR, and clean up some of the comments and TODOs.

Reply:
This makes to me, and I included it in the version I just pushed.

Any opinions on the name of the config value? I made it a top-level key called `max_line_len` and it currently gets parsed to a `usize`.

Reply:
`line_wrapping_width`?

Reply:
I think we can add some sensible defaults for the languages.toml like rust 100.

Reply:
Hmm I'll have to look into that. Any tips on where those defaults might go? I was a bit confused by the code that processes the `languages.toml` file.

Reply:
Go to
```toml
[[language]]
name = "rust"
max_line_len = 100
```
That is just an example, I think the variable name may not the that.